app/views/curation_concerns/multi_volume_works/_members.html.erb

```diff
@@ -15,7 +15,11 @@
           <tr>
             <td class="col-xs-3"><%= render_thumbnail_tag res %></td>
             <td>
-              <%= link_to res.title, main_app.curation_concerns_member_scanned_resource_path(@presenter.id, res.id) %>
+              <% if @presenter.model_name.singular == "scanned_resource" %>
```

Comment:
Is there a better way to check if it's a ScannedResource?  Not sure if checking the presenter class is any better...